docs/development/01-contribution.md

```diff
@@ -0,0 +1,46 @@
+---
+id: contribution
+title: Contribution guidelines
+---
+
+As a decentralized network, KILT depends on the support of its community. There are many ways to contribute to KILT protocol and the products and services built on it. 
+
+If you are interested in contributing but unsure how to begin, start in our [Clan KILT Discord](https://discord.gg/7uyfMXh6AT) channel. The active developers, which include the team that originally developed KILT, and the wider KILT community are engaged there, and can:
+
+- provide feedback on proposals or ideas
+- discuss possible use-cases or feature requests
+- make suggestions for non-technical contributions, including events, writing, or business models
+- answer questions about the protocol, services and products.
+
+## Feature Requests
+
+For the integrity of the protocol, requesting a feature requires approval from KILT governance. The request should follow the guidelines below:
+
+ - Begin a discussion with the community to ensure that they agree that it adds value to the KILT protocol in a meaningful way, supporting its goals
+ - Give your proposal a clear title
+ - Open an issue on the corresponding repository
+ - Provide a written outline of the feature request for discussion
+
+## Feature Proposals
+
+A proposal must be made before moving ahead, see above. At the end of the discussion, the proposal needs the approval of the community, KILT Technical Committee, governance, or relevant project team (depending on the subject) to progress to the next stage. If approved, acceptable deliverables will be agreed with the proposer, and the active developers or appropriate teams can start working on the proposal deliverables.
```

Comment:
I still think it is confusing and mixing things up. AFAICS, this section should just be about proposal, which the proposer will have to do and the previous section is just for requesting features, which kilt will have to do...
I think it needs a bit more thought on how to actually structure these sections.

Reply:
Updated